### ISSUE TITLE
Expose options for spin/loading icon

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-web-map-core#77]
https://github.com/Greenstand/treetracker-web-map-core/blob/1fbaa63c264711af9217c5bbc6d066de032c53e5/src/Map.js#L365-L366

So for clients which are going to use the web map core, they can set up the spin behaviors via:
new Map({
disableSpinIcon: boolean,
slowThreshold: [ms],
showLoadingThreshold: [ms],
})

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-core#treetracker-web-map-core) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

**Comment by SuokN** [comment]
Is disableSpinIcon need for disabling message "Trees grow slower than this map loads, be patient..."? 
If it is still actual i want to do this issue.

**Comment by dadiorchen** [comment]
Yes, exactly, and the option for the value of those time setting.

**Comment by keyy123** [comment]
@dadiorchen  @SuokN  Is this ticket closed?

**Comment by SuokN** [comment]
> @dadiorchen @SuokN Is this ticket closed?
PR for this ticket is still in PRs List,  that's why it is not closed. When someone checks  the PR, it will be closed or returned. So I think you can't take it now.

**Comment by keyy123** [comment]
Okay, Thank you for letting me know @SuokN.
